libraries/config/mechanical_material_data.hpp

```diff
@@ -35,26 +35,46 @@ struct mechanical_material_data<T, 1> final {
 };
 
 template<std::floating_point T>
-struct mechanical_material_data<T, 2> final {
-    static constexpr std::string_view Prefix = "mechanical";
+class mechanical_material_data<T, 2> final {
+    void read_elasticity_parameters(std::array<T, 2>& parameter, const nlohmann::json& config, std::string name) {
+        const auto& conf = config[name];
+        if (config.is_number()) {
```

Comment:
Перенести с 40-й строчки в if
```suggestion
        if (const auto& conf = config[name]; config.is_number()) {
```

Reply:
Перенес